src/Compilers/VisualBasic/Portable/Emit/NamedTypeSymbolAdapter.vb

```diff
@@ -353,6 +353,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             End If
         End Function
 
+        Private Function ShouldIncludePrivateField(field As FieldSymbol) As Boolean
```

Comment:
>ShouldIncludePrivateField [](start = 25, length = 25)

The function doesn't check accessibility. Is it really necessary to assume the field is private?

Reply:
"IsWithEventsField"?

---
In reply to: [569594611](https://github.com/dotnet/roslyn/pull/50930#discussion_r569594611) [](ancestors = 569594611)